.github/workflows/swaggerHub-generate-sync.yaml

```diff
@@ -50,14 +54,14 @@ jobs:
 
       - name: Verify Swagger file
         run: |
-            if [ ! -f swagger.yaml ]; then
-              echo "swagger.yaml not found!"
-              exit 1
-            fi
+          if [ ! -f swagger.yaml ]; then
+            echo "swagger.yaml not found!"
+            exit 1
+          fi
 
       - name: Sync with SwaggerHub
         run: |
-          curl -X POST "https://api.swaggerhub.com/apis/${{ inputs.owner }}/${{ inputs.api_name }}?isPrivate=true" \
+          curl -X POST "https://api.swaggerhub.com/apis/${{ inputs.owner }}/${{ inputs.api_name }}/${{ inputs.release_version }}?isPrivate=true" \
```

Comment:
As I can see in another solutions from STEM, [here](https://github.com/stetind/stem-api/blob/main/.github/workflows/create-docs.yaml), the version used as query parameter not as a path parameter.

Please see the documentation [here](https://app.swaggerhub.com/apis-docs/swagger-hub/registry-api/1.0.67#/APIs/saveDefinition) `/apis/{owner}/{api}` 

Also in the same solution, we can see an alternative idea about the communication with the swagerhub. (if we have time we can check the alternative, otherwise we will see it in the future)